myia/parser.py

```diff
@@ -769,6 +783,14 @@ def process_Assign(self, block: "Block", node: ast.Assign) -> "Block":
 
         return block
 
+    def process_AnnAssign(self, block: "Block", node: ast.AnnAssign) -> "Block":
+        """Process an annotated assignment."""
+        anf_node = self.process_node(block, node.value)
+        # Get type annotation for variable.
+        anf_node.annotation = astunparse.unparse(node.annotation).strip()
```

Comment:
I think it would be preferable to evaluate the annotations (in the function's environment), rather than store them as strings. Basically something like `eval(annotation, self.function.__globals__)`. I think you can do that from the AST so it might not be necessary to unparse. Also, there should be an informative error pointing to the annotation if the eval fails.